js/freewheel.js

```diff
@@ -261,9 +263,39 @@ OO.Ads.manager(function(_, $) {
         ].join('')
       ];
 
-      // TODO: Implement a timeout here in case fw_onAdRequestComplete doesn't get called - PBI-731
+      // Set up function to log error if ad response is null after maximum duration allowed for ad request to respond
+      // has exceeded.
+      var adRequestTimeout = _.bind(function(){
```

Comment:
Is it possible to move this function to be defined out side. In general we are trying to not define functions within functions. (Easier to debug.)